### DISCUSSION TITLE
Kidnapping

### DISCUSSION THREAD

**Discussion by someguythatplaysbarotrauma** [FakeFishGames/Barotrauma#8532] [Ideas and suggestions]
You should be able to kidnap people from outposts by paralyzing them, handcuffing them, and then bringing them onto your ship.

**Comment by AFK-TankyMOFO** [comment]
But... What does that accomplish? What's the benefit?

**Reply by someguythatplaysbarotrauma** [reply]
Fun, that's the benefit... and a duffel bag!

**Reply by mLuby** [reply]
If [By the Book](https://barotraumagame.com/wiki/By_the_Book) worked on _all_ hostile handcuffed humans alive on your sub, not just ones who [_started the round hostile_](https://github.com/Regalis11/Barotrauma/blob/6cc100d98cc99e758f4ea0750528a2aa68429c0a/Barotrauma/BarotraumaShared/SharedSource/Characters/Talents/Abilities/CustomAbilities/CharacterAbilityByTheBook.cs#L22) (i.e. pirates and squatters), you could gain money and experience from snatching people. I assume it's intentional that you cannot do this.

**Reply by James-E-A** [reply]
> ... and a duffel bag!

FYI: haul all the Bandit corpses onto your sub during Abandoned Outpost rounds as an easy source of these (don't forget that they _drop_ their equipped item upon death, so watch out for neglected revolvers if you meant to sell their stuff)

**Comment by James-E-A** [comment]
Could be a fun way to burn 120 reputation in exchange for a crewmember

**Reply by James-E-A** [reply]
@someguythatplaysbarotrauma FYI, you can use chloral hydrate if you're low on paralyzant (or don't want to waste anaparalyzant)
